specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -64,7 +64,12 @@ const isDarkMode = ({
   isRedirecting,
 }: PreferencesVisibilityContext): boolean => isDarkMode || isRedirecting;
 
-const altKeyName = globalThis.navigator?.appVersion.includes('Mac')
+/**
+ * REFACTOR: navigator.platform is deprecated. Implement better feature
+ * detection for browsers using Mac.
+ * See https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform
+ */
+const altKeyName = globalThis.navigator?.platform.startsWith('Mac')
```

Comment:
this is exactly the use case that was quoted by you from the MDN article, so probably using navigator.platform in the right choice in this case?